File: practice/01/solution.cc
std::vector<string> Search (std::vector<long> A, std::vector<long> Q)
{
  std::vector<string> R;
  std::sort(A.begin(), A.end());
  for (auto const &q : Q)
  {
    if (std::binary_search(A.begin(), A.end(), q))
    {
      R.push_back("yes");
    }
    else
    {
      R.push_back("no");
    }
  }
  return R;
}

File: practice/01/solution.py
#!/usr/bin/env python
# coding: utf-8



from bisect import bisect_left 

def Search(A, Q):
    # Write your code here
    output = []
    A.sort()
    for element in Q:
        i = bisect_left(A, element)
        if i != len(A) and A[i] == element:
            output.append('yes')
        else:
            output.append('no')
    return(output)

